dev/dependencyList

```diff
@@ -133,24 +133,21 @@ metrics-json/4.2.8//metrics-json-4.2.8.jar
 metrics-jvm/4.2.8//metrics-jvm-4.2.8.jar
 netty-all/4.1.73.Final//netty-all-4.1.73.Final.jar
 netty-buffer/4.1.73.Final//netty-buffer-4.1.73.Final.jar
-netty-codec-dns/4.1.74.Final//netty-codec-dns-4.1.74.Final.jar
-netty-codec-http/4.1.77.Final//netty-codec-http-4.1.77.Final.jar
-netty-codec-http2/4.1.77.Final//netty-codec-http2-4.1.77.Final.jar
-netty-codec-socks/4.1.77.Final//netty-codec-socks-4.1.77.Final.jar
+netty-codec-dns/4.1.73.Final//netty-codec-dns-4.1.73.Final.jar
+netty-codec-http/4.1.73.Final//netty-codec-http-4.1.73.Final.jar
+netty-codec-http2/4.1.73.Final//netty-codec-http2-4.1.73.Final.jar
+netty-codec-socks/4.1.73.Final//netty-codec-socks-4.1.73.Final.jar
 netty-codec/4.1.73.Final//netty-codec-4.1.73.Final.jar
 netty-common/4.1.73.Final//netty-common-4.1.73.Final.jar
-netty-handler-proxy/4.1.77.Final//netty-handler-proxy-4.1.77.Final.jar
+netty-handler-proxy/4.1.73.Final//netty-handler-proxy-4.1.73.Final.jar
 netty-handler/4.1.73.Final//netty-handler-4.1.73.Final.jar
-netty-resolver-dns/4.1.74.Final//netty-resolver-dns-4.1.74.Final.jar
+netty-resolver-dns/4.1.73.Final//netty-resolver-dns-4.1.73.Final.jar
 netty-resolver/4.1.73.Final//netty-resolver-4.1.73.Final.jar
 netty-tcnative-classes/2.0.46.Final//netty-tcnative-classes-2.0.46.Final.jar
 netty-transport-classes-epoll/4.1.73.Final//netty-transport-classes-epoll-4.1.73.Final.jar
-netty-transport-classes-kqueue/4.1.73.Final//netty-transport-classes-kqueue-4.1.73.Final.jar
 netty-transport-native-epoll/4.1.73.Final/linux-aarch_64/netty-transport-native-epoll-4.1.73.Final-linux-aarch_64.jar
 netty-transport-native-epoll/4.1.73.Final/linux-x86_64/netty-transport-native-epoll-4.1.73.Final-linux-x86_64.jar
-netty-transport-native-kqueue/4.1.73.Final/osx-aarch_64/netty-transport-native-kqueue-4.1.73.Final-osx-aarch_64.jar
-netty-transport-native-kqueue/4.1.73.Final/osx-x86_64/netty-transport-native-kqueue-4.1.73.Final-osx-x86_64.jar
```

Comment:
Since users may less likely to run Kyuubi on macOS for production, macOS-specific Netty native libraries(kqueue) have been removed from the binary release tarball. This change will not break anything on macOS, Netty will fall back to NIO mode if kqueue libraries unavailable.